pkg/cloudprovider/launchtemplate.go

```diff
@@ -251,15 +252,16 @@ func (p *LaunchTemplateProvider) Invalidate(ctx context.Context, ltName string)
 	defer p.Unlock()
 	defer p.cache.OnEvicted(p.cachedEvictedFunc(ctx))
 	p.cache.OnEvicted(nil)
-	logging.FromContext(ctx).Debugf("Invalidating launch template \"%s\" in the cache because it no longer exists", ltName)
+	logging.FromContext(ctx).Debugf("invalidating launch template in the cache because it no longer exists")
```

Comment:
Is there a reason we aren't logging with the launch template name here?

Reply:
The name is passed inside the context